@xen-orchestra/proxy-cli/src/index.js

```diff
@@ -85,13 +86,33 @@ ${pkg.name} v${pkg.version}`
     request.port = port
   }
 
-  const lines = pumpify.obj(await hrp.post(request), split2())
+  const response = await hrp.post(request)
+
+  const { stdout } = process
+  const { httpVersion, statusCode, statusMessage } = response
+  stdout.write(
+    inspect(`HTTP/${httpVersion} ${statusCode} ${statusMessage}`, {
+      colors: true,
+      depth: null,
+    })
+  )
+  stdout.write('\n')
+
+  const responseType = contentType.parse(response).type
+  stdout.write(inspect(`Content-Type: ${responseType}`, { colors: true, depth: null }))
+  stdout.write('\n\n')
+
+  if (responseType === 'application/octet-stream') {
+    stdout.write('Binary data')
+    return
+  }
```

Comment:
This does not make sense in production, it could be converted to `assert`s though.

Reply:
Do mean to replace the above block by an `assert`?

Reply:
Yes, as I suggested in another comment :slightly_smiling_face: